binstar_client/utils/__init__.py

```diff
@@ -149,30 +150,56 @@ def load_token(url):
         token = None
     return token
 
-def get_binstar(args=None, cls=None):
+
+def get_server_api(token=None, site=None, log_level=logging.INFO, cls=None):
+    """
+    Get the anaconda server api class
+    """
+
     if not cls:
         from binstar_client import Binstar
         cls = Binstar
-    config = get_config(remote_site=args and args.site)
+    config = get_config(remote_site=site)
     url = config.get('url', DEFAULT_URL)
-    if getattr(args, 'log_level', 0) >= logging.INFO:
+
+    if log_level >= logging.INFO:
         sys.stderr.write("Using Anaconda Cloud api site %s\n" % url)
-    if args and args.token:
+    if token:
         log.debug("Using token from command line args")
-        token = args.token
     elif 'BINSTAR_API_TOKEN' in os.environ:
         log.debug("Using token from environment variable BINSTAR_API_TOKEN")
         token = os.environ['BINSTAR_API_TOKEN']
     elif 'ANACONDA_API_TOKEN' in os.environ:
         log.debug("Using token from environment variable ANACONDA_API_TOKEN")
         token = os.environ['ANACONDA_API_TOKEN']
+
     else:
         token = load_token(url)
 
-
     verify = config.get('verify_ssl', True)
     return cls(token, domain=url, verify=verify)
 
+def get_binstar(args=None, cls=None):
+    """
+    DEPRECATED METHOD,
+
+
+    use `get_server_api`
+    """
+
+    warnings.warn(
+      'method get_binstar is depricated, please use `get_server_api`',
+      DeprecationWarning
+    )
+
+    token = getattr(args, 'token', None)
+    log_level = getattr(args, 'log_level', 0)
+    site = getattr(args, 'site', 0)
```

Comment:
is `0` the correct default for `site`? In `get_server_api`, the default is `None`